namespaces/cloud-platform-live-0.k8s.integration.dsd.io/kube-ops/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    application: kube-ops
+    version: v0.8.1
+  name: kube-ops
+  namespace: kube-ops
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      application: kube-ops
+  template:
+    metadata:
+      labels:
+        application: kube-ops
+        version: v0.8.1
+    spec:
+      serviceAccount: kube-ops
+      containers:
+      - name: service
+        # see https://github.com/hjacobs/kube-ops-view/releases
+        image: hjacobs/kube-ops-view:0.8.1
```

Comment:
Ideally we want to manage the image but I'm happy for this to go in as-is right now. cc @jasonBirchall